From 184433236a23351af6db800d5f93459bc998ac00 Mon Sep 17 00:00:00 2001
From: Erfan Abdi <erfangplus@gmail.com>
Date: Fri, 14 Jan 2022 17:45:57 +0330
Subject: [PATCH] volla: Disable RecoveryUpdater page

---
 .../setupwizard/util/SetupWizardUtils.java       | 16 +---------------
 1 file changed, 1 insertion(+), 15 deletions(-)

diff --git a/src/org/lineageos/setupwizard/util/SetupWizardUtils.java b/src/org/lineageos/setupwizard/util/SetupWizardUtils.java
index fb89a05..03f65fc 100644
--- a/src/org/lineageos/setupwizard/util/SetupWizardUtils.java
+++ b/src/org/lineageos/setupwizard/util/SetupWizardUtils.java
@@ -111,21 +111,7 @@ public class SetupWizardUtils {
     }
 
     public static boolean hasRecoveryUpdater(Context context) {
-        boolean fileExists = new File(UPDATE_RECOVERY_EXEC).exists();
-        if (!fileExists) {
-            return false;
-        }
-
-        boolean featureHidden = false;
-        try {
-            PackageManager pm = context.getPackageManager();
-            Resources updaterResources = pm.getResourcesForApplication(UPDATER_PACKAGE);
-            int res = updaterResources.getIdentifier(
-                    CONFIG_HIDE_RECOVERY_UPDATE, "bool", UPDATER_PACKAGE);
-            featureHidden = updaterResources.getBoolean(res);
-        } catch (PackageManager.NameNotFoundException | Resources.NotFoundException ignored) {
-        }
-        return !featureHidden;
+        return false;
     }
 
     public static boolean isRadioReady(Context context, ServiceState state) {
-- 
2.29.2

