From 4a4b81fe20da184d8f8130d79fdcfb1b6d474b0d Mon Sep 17 00:00:00 2001
From: Volla BuildBot <build@volla.com>
Date: Tue, 2 Jun 2026 10:52:19 +0000
Subject: [PATCH] Add volla cert placeholder

---
 files/735ba42f.0 | 9 +++++++++
 1 file changed, 9 insertions(+)
 create mode 100644 files/735ba42f.0

diff --git a/files/735ba42f.0 b/files/735ba42f.0
new file mode 100644
index 0000000..c1b1097
--- /dev/null
+++ b/files/735ba42f.0
@@ -0,0 +1,9 @@
+-----BEGIN CERTIFICATE-----
+MIIBSTCB8KADAgECAgEBMAoGCCqGSM49BAMCMBwxGjAYBgNVBAMMEVZvbGxhIEZp
+cmV3YWxsIENBMB4XDTI2MDYwMjAwMjUxNVoXDTM2MDUzMDAwMjUxNVowHDEaMBgG
+A1UEAwwRVm9sbGEgRmlyZXdhbGwgQ0EwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNC
+AARh0sD7PX7oygJbLEuCGLkDpBngTZl5KZAq/Hf3rnHn+6ZHSX7XrA0r1VPtIlBR
+kkRi5BM/S2izSyhUheaU4fhyoyMwITAPBgNVHRMBAf8EBTADAQH/MA4GA1UdDwEB
+/wQEAwIBBjAKBggqhkjOPQQDAgNIADBFAiATcYi8yYpeAupIjcWTLZfvkenvGWfO
+ojmamdeNReEQLwIhAMm3VyOPvMzbNr2rJ6HP7OMhtgNEJXNb738h6KbSx/rB
+-----END CERTIFICATE-----
-- 
2.34.1

